src/qibo/hamiltonians.py

```diff
@@ -186,3 +188,129 @@ def MaxCut(nqubits, dense=True):
     if dense:
         return ham.dense
     return ham
+
+
+from qibo.symbols import X as tspX
+from qibo.symbols import Y as tspY
+from qibo.symbols import Z as tspZ
+class TSP:
+    """
+    This is a TSP class that enables us to implement TSP according to
```

Comment:
I would explain the TSP acronym the first time is used:
```suggestion
    This is a TSP (Travelling Salesman Problem) class that enables us to implement TSP according to
```